deepcell_label_utils/utils.py

```diff
@@ -25,13 +25,14 @@
 # ==============================================================================
 """Label utils"""
 
-import json
-import geojson
+# import json
+# import geojson
 import numpy as np
 import cv2
 
-from shapely.geometry import MultiPolygon, Polygon, Point, mapping
+from shapely.geometry import MultiPolygon, Polygon, Point  # , mapping
 from collections import defaultdict
+from skimage.measure import regionprops
```

Comment:
Please reorganize this imports according to the guidelines below in another comment